lua/neotest-busted/init.lua

```diff
@@ -307,7 +315,7 @@ function BustedNeotestAdapter.create_test_command(paths, options)
     end
 
     return {
-        nvim_command = compat.loop.exepath(),
+        nvim_command = config.no_nvim and busted_command or compat.loop.exepath(),
```

Comment:
I think we should change `nvim_command` to just `command` as it isn't necessarily a neovim command anymore.